pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -92,6 +95,12 @@ var updateDeploymentAndWait = k8sutil.UpdateDeploymentAndWait
 
 // Start begins the process of running a cluster of Ceph mgrs.
 func (c *Cluster) Start() error {
+	// Validate pod's memory if specified
+	err := opspec.CheckPodMemory(c.resources, cephMgrPodMinimumMemory)
+	if err != nil {
+		return fmt.Errorf("%v", err)
```

Comment:
for future reference, you could just `return err` if an additional message isn't being appended